Specifications/Tool/com.apple.compilers.pbxcp.xcspec

```diff
@@ -11,7 +11,7 @@
     Type = Tool;
     Identifier = com.apple.compilers.pbxcp;
     Name = "Copy Utility";
-    CommandLine = "builtin-copy [options] [inputs] [output]";
+    CommandLine = "builtin-copy [options] [inputs] $(OutputPath:dir)";
```

Comment:
I think this was wrong, but in a different way: `CopyResolver.cpp` already adds the output directory to the end. This can just have the options:

     ExecPath = "builtin-copy";